js/companyInfo.js

```diff
@@ -0,0 +1,98 @@
+let urlParams = new URLSearchParams(window.location.search);
+const symbol = urlParams.get("symbol");
+const companyProfileUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/`;
+const spinner = document.getElementById("spinnerForCompany");
+const myChart = document.getElementById("myChart");
+
+async function fetchCompanyData(companyProfileUrl) {
```

Comment:
this function not onlt fetch the data, so, maybe think about a name that is more precise for it?